autoscaler/controllers/actions/addclusterinfo_controller.go

```diff
@@ -45,21 +44,28 @@ func (r *AddClusterInfoReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		return ctrl.Result{}, client.IgnoreNotFound(err)
 	}
 
-	processor, err := r.convertToProcessor(action)
+	// Migrate to odigosv1.Action
+	migratedActionName := v1.ActionMigratedLegacyPrefix + action.Name
+	odigosAction := &v1.Action{}
+	err = r.Get(ctx, client.ObjectKey{Name: migratedActionName, Namespace: action.Namespace}, odigosAction)
 	if err != nil {
-		r.ReportReconciledToProcessorFailed(ctx, action, FailedToTransformToProcessorReason, err.Error())
```

Comment:
worth noting, in the new code I'm not updating the legacy action with its status anymore. This is because update conflicts caused a hotloop that I wasn't able to figure a good way around (if trying to update the action failed due to a conflict, then trying to update the status would still be a conflict, and requeues never seemed to sort it out)